pkgs/applications/window-managers/xborders/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, python3Packages, fetchFromGitHub, libwnck, gtk3, libnotify, wrapGAppsHook, gobject-introspection }:
+
+with python3Packages; buildPythonPackage rec {
+  pname = "xborders";
+  version = "3.4"; # in version.txt
+  src = fetchFromGitHub {
+    owner = "deter0";
+    repo = "xborder";
+    rev = "d840ed649e92b7ad399cdb1a30aeca7d09f2c08f";
+    sha256 = "sha256-Q2WWXekUL/OOr/OBqJOZbAHaWURbP+coBVp206uPSNY=";
+  };
+  buildInputs = [ libwnck gtk3 libnotify ];
+  nativeBuildInputs = [ wrapGAppsHook gobject-introspection ];
+  propagatedBuildInputs = [ pycairo requests pygobject3 ];
+  preBuild = ''
+    sed '52,58!d' ${pname}
+    sed -i '51 a\    return ${version}' ${pname}
+    cat >setup.py <<'EOF'
```

Comment:
Shouldn't it be upstreamed?

Reply:
Maybe? I'll make a PR upstream and see what the owner says.

Reply:
https://github.com/deter0/xborder/pull/55

Reply:
Merged.